k6/scenarios/auth.scenarios.ts

```diff
@@ -0,0 +1,124 @@
+import { check, sleep } from "k6";
```

Comment:
Should we keep scenarios file to our specific module folders?

Reply:
That, we will have to look into it.

Reply:
It's okay, we can decide this later when we will add more load scenarios.